science/mpich/Portfile

```diff
@@ -155,16 +179,22 @@ configure.optflags-delete   -O2 -Os
 configure.cppflags-delete   -I${prefix}/include
 configure.ldflags-delete    -L${prefix}/lib
 
-if {${subport} != ${name}} {
+if {${name} ne ${subport}} {
```

Comment:
This is another place that should exclude the obsolete subports.

Reply:
Done